### PR TITLE
Admin delete sighting

### PR DESCRIPTION
**Pull request by RoshniCh**
added the delete end point for admin. this will change the confirm state of the id passed to the endpoint

### CODE REVIEW

**Review by littlemauli — COMMENTED** [review]
all good for me :-)

**Review by chloejonessoftwire — COMMENTED** [review]
looks good to me! :)

File: ClientApp/src/Api/apiClient.ts [hunk 1]
```diff
@@ -69,3 +69,12 @@ export async function fetchUnconfirmedSightings(): Promise<null | ListSightings>
   return await response.json();
 }
 
+export async function delSighting(Id:number) {
```
Comment (ChristianMcc) [inline ClientApp/src/Api/apiClient.ts:72]: can we call this deleteSighting please? It's a big nitpick and not much difference but just to emphasis clear function names :)

File: Controllers/AdminController.cs [hunk 2]
```diff
@@ -23,5 +23,14 @@ public Sighting GetSightingDetails([FromRoute] int id)
             var sighting = _sightings.SelectSightingById(id);
             return sighting;
         }
+
+        [HttpPost("deleteSighting/{id}")]
+        public Sighting DeleteSighting([FromRoute] int id)
+        {
+            var sighting = _sightings.SelectSightingById(id);
+            sighting.ConfirmState = ConfirmState.Deleted;
```
Comment (ChristianMcc) [inline Controllers/AdminController.cs:31]: Could we move this line into the repo method please

**Review by ChristianMcc — APPROVED** [review]
LGTM, 2 small nitpicks - I'll approve and just let you merge it in once they are done :)

**Pull request merged**